contracts/ServiceManagerToken.sol

```diff
@@ -31,32 +31,32 @@ contract ServiceManagerToken is GenericManager {
     uint96 public constant BOND_WRAPPER = 1;
     // Operator whitelist address
     address public operatorWhitelist;
+    // Service manager version number
+    string public constant VERSION = "1.1.0";
 
     /// @dev ServiceRegistryTokenUtility constructor.
     /// @param _serviceRegistry Service Registry contract address.
     /// @param _serviceRegistryTokenUtility Service Registry Token Utility contract address.
-    constructor(address _serviceRegistry, address _serviceRegistryTokenUtility) {
-        // Check for the zero address
+    constructor(address _serviceRegistry, address _serviceRegistryTokenUtility, address _operatorWhitelist) {
+        // Check for the Service Registry related contract zero addresses
```

Comment:
Since we can skip it here, and in general delete it by setting to `address(0)`

Reply:
possible combination: as disable operatorWhitelist  by default in deploy

Reply:
Yeah, but since we already give the possibility to SetCheck equal to false, I do not see the real motivation to further allow the DAO to disable the operatorWhitelist in this way. A governance vote would still be required to disable/enable it and in the eventuality, DAO wants to completely disable the operatorWhitelist a new manager contract can be considered.

Reply:
Disabling is still much cheaper than deploying a new contract